Feature request should be made at the org level (#1238)
home-assistant/companion.home-assistant: docs/troubleshooting/more-help.md

*   Visit the [Companion Apps channel](https://discord.com/channels/330944238910963714/1284965926336335993)
*   Post on the [iOS forums](https://community.home-assistant.io/c/mobile-apps/ios) or [Android forums](https://community.home-assistant.io/c/mobile-apps/android-companion).

**For bug reports**
*   Please raise an issue on the [iOS GitHub repository](https://github.com/home-assistant/iOS) or the [Android GitHub repository](https://github.com/home-assistant/android).

**For feature requests**

> * To request a new feature, open a discussion in the Home Assistant GitHub organization using the [Android discussions page](https://github.com/orgs/home-assistant/discussions/new?category=android) or the [iOS and macOS discussions page](https://github.com/orgs/home-assistant/discussions/new?category=ios-macos).

Reviewer: ```suggestion
* To request a new feature, open a discussion in the Home Assistant GitHub organization. Use the discussion page of the related platform:
  -  [Android discussions page](https://github.com/orgs/home-assistant/discussions/new?category=android)
  -  [iOS and macOS discussions page](https://github.com/orgs/home-assistant/discussions/new?category=ios-macos).
```
tweak
